MovieMVVM/MovieMVVMTests/DetailsMovieViewModelTests.swift

```diff
@@ -0,0 +1,82 @@
+// DetailsMovieViewModelTests.swift
+// Copyright © KarpovaAV. All rights reserved.
+
+@testable import MovieMVVM
+import XCTest
+
+/// Тест вью модели экрана деталей фильмов
+final class DetailsMovieViewModelTests: XCTestCase {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let idNumber = 315_162
+        static let fooText = "foo"
+        static let intNumberOne = 1
+        static let bazText = "baz"
+        static let barText = "bar"
+        static let doubleNumberOne = 1.0
+    }
+
+    // MARK: - Private Properties
+
+    private var detailsMovieViewModel: DetailsMovieViewModelProtocol?
+    private var mockDataService = MockDataService()
+    private var mockImageService = MockImageService()
+    private var mockDetailsMovie = MovieDetails(
+        posterPath: Constants.bazText,
+        overview: Constants.bazText,
+        title: Constants.bazText,
+        rating: Constants.doubleNumberOne,
+        tagline: Constants.bazText,
+        releaseDate: Constants.bazText,
+        runtime: Constants.intNumberOne,
+        productionCountries: [ProductionCountries(name: Constants.bazText)],
+        id: Constants.idNumber
+    )
+    private var mockRecommendationMovieResponse =
+        RecommendationMovieResponse(movies: [RecommendationMovie(posterPath: Constants.fooText)])
+
+    // MARK: - Public Methods
+
+    override func setUp() {
```

Comment:
```suggestion
    override func setUp() {
```
Надо super добавлять